relayer/main/main.go

```diff
@@ -534,3 +535,125 @@ func initializeMetrics() (prometheus.Gatherer, prometheus.Registerer, error) {
 	}
 	return gatherer, registry, nil
 }
+
+func initializeConnectionsAndCheckStake(
```

Comment:
`initializeConnectionsAndCheckStake`, `connectToNonPrimaryNetworkPeers`, and `connectToPrimaryNetworkPeers` may make more sense to live in a non-main package specific to the relayer, but not a huge deal either way.

Reply:
Yeah I thought about it originally, but they are only called as part of initialization at app startup. I am happy to move them but any idea on the naming? I don't think we need a new package for it since it would define no structs, methods or interfaces. 

Maybe `network_utils.go` in the `relayer` package?

Reply:
Sounds good to me 👍